web-app/js/portal/details/SubsetItemsWrapperPanel.js

```diff
@@ -74,6 +89,7 @@ Portal.details.SubsetItemsWrapperPanel = Ext.extend(Ext.Panel, {
         this.spinnerToolItem = {
             id: 'spinnerToolItem',
             styles: 'fa-spin fa-spinner',
+            hidden: !layer.loading,
```

Comment:
If `undefined` or `null`, then the spinner will be hidden, which I think is correct.  I prefer this style (of "truthiness") but it's just a preference.